shader/writeback.wgsl

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: Apache-2.0 OR MIT OR Unlicense
```

Comment:
Why is this shader necessary? Can't we use [`wgpu::CommandEncoder::copy_buffer_to_texture`](https://docs.rs/wgpu/latest/wgpu/struct.CommandEncoder.html#method.copy_buffer_to_texture) or `Command::UploadImage` to perform the upload?

Reply:
That can't be used with just a texture view.